test/tests/diff.spec.ts

```diff
@@ -15,7 +15,6 @@ const V3R7 = processInSpecProfile(fs.readFileSync('test/sample_data/inspec/json/
 const V3R6 = processInSpecProfile(fs.readFileSync('test/sample_data/inspec/json/rhel-7-v3r6-mini-profile.json', 'utf-8'));
 const V3R8 = processXCCDF(fs.readFileSync('test/sample_data/xccdf/input/STIG/rhel-7-v3r8-mini-sample-xxcdf.xml', 'utf-8'), false, 'rule');
 
-
```

Comment:
On line 27 of this file we have a writeFileSync which would write the expected file every time. Do we want to comment that out / remove that so we are testing against a constant baseline?

Reply:
The file we are writing is the diff between the two files, than we read it back. We can change to this:
const expected = JSON.parse(JSON.stringify(diffProfile(V1R2, V1R3, thisLogger), null, 2))